pkg/monitor/api.go

```diff
@@ -7,6 +7,7 @@ import (
 	"sync"
 	"time"
 
+	e2epromclient "github.com/openshift/origin/test/extended/prometheus/client"
```

Comment:
This is not allowed - pkg/monitor may not take dependencies on deep test packages in general.